resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/VolcanoFeatureStep.scala

```diff
@@ -77,3 +70,8 @@ private[spark] class VolcanoFeatureStep extends KubernetesDriverCustomFeatureCon
     SparkPod(k8sPod, pod.container)
   }
 }
+
+private[spark] object VolcanoFeatureStep {
+  val POD_GROUP_ANNOTATION = "scheduling.k8s.io/group-name"
+  val POD_GROUP_TEMPLATE_FILE_KEY = "spark.kubernetes.scheduler.volcano.podGroupTemplateFile"
```

Comment:
Since this is only applicable for `volcano` scheduler and `VolcanoFeatureStep`, this should be here.

Reply:
+1

Reply:
+1, I'm fine with this, then I just need to doc it in volcano section, right?

Reply:
Yes, right~ Now, it's time to document here. Could you write some, @Yikun ?
- https://spark.apache.org/docs/latest/running-on-kubernetes.html